src/test/java/com/notably/model/block/BlockTreeItemTest.java

```diff
@@ -0,0 +1,23 @@
+package com.notably.model.block;
+
+import static com.notably.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.scene.control.TreeItem;
+
+public class BlockTreeItemTest {
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new BlockTreeItemImpl((Block) null));
+        assertThrows(NullPointerException.class, () -> new BlockTreeItemImpl((TreeItem<Block>) null));
+    }
+
+    @Test
+    public void is_root_returnsTrue() {
```

Comment:
Following our mod's [naming convention](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html), let's rename this to `isRootBlock_rootBlock_returnsTrue`?